manim/mobject/geometry.py

```diff
@@ -598,13 +632,13 @@ def construct(self):
 
     def __init__(
         self,
-        point=ORIGIN,
+        point: np.ndarray = ORIGIN,
         radius: float = DEFAULT_DOT_RADIUS,
-        stroke_width=0,
-        fill_opacity=1.0,
-        color=WHITE,
+        stroke_width: int = 0,
```

Comment:
This is showing up in a few places too. We'll want the defaults to be 0.0 to indicate that they're floats.

Reply:
Yep like I mentioned monkey type is good, but some things it just can't know. I've left it raw for demo purposes.